src/CMakeLists.txt

```diff
@@ -145,6 +145,28 @@ find_package(Lua REQUIRED)
 
 find_package(GMP REQUIRED)
 
+OPTION(ENABLE_POSTGRESQL "Enable PostgreSQL backend" FALSE)
```

Comment:
This should be true, like with every other backend, that way PostgreSQL support will automatically be included if it's available.